docs/Docker.md

```diff
@@ -20,11 +20,12 @@ The following steps can be used to setup Stringer on Docker, using a Postgres da
 docker run --detach \
     --name stringer-postgres \
     --restart always \
-    --volume /srv/stringer/data:/var/lib/postgresql/data \
+    --volume ./postgres:/var/lib/postgresql/data \
```

Comment:
Can you explain this change to me? I'm not at all familiar with docker, so not sure why this would change.

Reply:
I thought it would be better to gives to people a command which isn't install things all around your disks. Like that a postgres folder will be created to handle the docker volume at the current working directory.

I can revert this part if you want !

Reply:
I thought it would be cleaner to create the volume on current working dir, where the user will run the command, it simplify the install, and if the user want remove stringer from his instance he just have to remove the folder.

Reply:
Existing users will need to move their directory from `/srv/stringer/data` to `./postgres`, yeah. I think this is a better default, though.

Reply:
I noticed in the docker-compose that the default dir there is `~/stringer` which imo is an even better default.